### DISCUSSION TITLE
Confused with Docker Installation

### DISCUSSION THREAD

**Discussion by jxs714** [pretix/pretix#2585] [Q&A]
Hey guys,

Still trying to get the hang of understanding docker. I only installed the docker standalone on Plesk Control Panel Docker, and when loading my domain for the software I am getting this error when loading the page: Bad Request. We were unable to parse your request.

and the error I am getting in the logs is: consumer: Cannot connect to amqp://guest:**@127.0.0.1:5672//: [Errno 111] Connection refused.

Is there anyway I can get help here to get the ball rolling to use the software soon?

**Comment by raphaelm** [comment]
I think there are multiple things going on here, the "bad request" might indicate that your `url` value in `pretix.cfg` (which you might be lacking) does not match the `Host` header of your HTTP request. The error in the logs indicates that redis is not running or not reachable.

I don't think it's possible to install pretix *that* simple through Plesk, unfortunately, and I have never used Plesk before and am unable to provide more extensive support.
